mysql-test/suite/rpl/t/rpl_percona_bug995624.test

```diff
@@ -0,0 +1,46 @@
+source include/master-slave.inc;
+source include/not_gtid_enabled.inc;
+
+connection slave;
+set global userstat=ON;
+--echo Rows_updated on slave - starting situation
+select ROWS_UPDATED from information_schema.client_statistics;
```

Comment:
This is what I am doing, but in a slightly different way. I record number of Rows_updated on master and slave, and of course compare it to the value stored in result file - i.e. 0. Then after the inserts/updates/deletes I am checking Rows_updated on master and slave once again. The recorded value in result file is 13. It should be the same on master and slave (at least for the tables that are replicated). In other words I am checking how much Rows_updated on master and slave increased by means of comparing it with the value stored in result file, i.e. 13.